### PR TITLE
[Worker] Enable cilium network policy for code upload evaluation

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR - 

- [x] Enables cilium network policy in code upload evaluation setup to only allow connections from job pods to EvalAI and `archive.ubuntu.com` (to setup ubuntu packages for init-container).
- [x] Add additional log statements in worker setup 

Changes verified using worker setup on EC2 instance.

### CODE REVIEW

File: scripts/workers/code_upload_worker_utils/network_policies.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ spec:
       {}
   egress:
   - toFQDNs:
+    - matchName: archive.ubuntu.com
```
Comment (RishabhJain2018) [inline scripts/workers/code_upload_worker_utils/network_policies.yaml:11]: Different values for the same parameter name `matchName`?

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

**Comment by RishabhJain2018** [comment]
Also, what was the issue with Cilium initially?

**Comment by Ram81** [comment]
@RishabhJain2018 the cilium issue comes up when we deploy workers on fargate. This PR is tested by running a worker manually. I need to merge this PR to test it on fargate.

**Comment by RishabhJain2018** [comment]
Got it!

File: scripts/workers/code_upload_worker_utils/network_policies.yaml (hunk 1 above)
Comment (RishabhJain2018) [reply scripts/workers/code_upload_worker_utils/network_policies.yaml:11]: @Ram81 ?

Comment (Ram81) [reply scripts/workers/code_upload_worker_utils/network_policies.yaml:11]: @RishabhJain2018 we are adding multiple entries for `matchName` to allow access to different FQDNs. We need access to `archive.ubuntu.com` for our pre-start hook (the hook that updates submission evaluation started at)

Comment (RishabhJain2018) [reply scripts/workers/code_upload_worker_utils/network_policies.yaml:11]: I see..okay.

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**